packages/adapters/src/adapters/Cornerstone/ContourRoi.js

```diff
@@ -0,0 +1,82 @@
+import { utilities } from "dcmjs";
+
+import MeasurementReport from "./MeasurementReport";
+import CORNERSTONE_4_TAG from "./cornerstone4Tag";
+
+const { Polyline: TID300Polyline } = utilities.TID300;
+
+class ContourRoi {
```

Comment:
How does this differ from FreehandROI?  Would like to end up with just a single representation for both types if we can figure that out.